library/alloc/src/vec.rs

```diff
@@ -201,6 +201,26 @@ use crate::raw_vec::RawVec;
 /// can be slow. For this reason, it is recommended to use [`Vec::with_capacity`]
 /// whenever possible to specify how big the vector is expected to get.
 ///
+/// A vector containing the elements `'a'` and `'b'` with capacity 4 can be
+/// visualized as:
+///
+/// ```text
+/// Stack       ptr      len  capacity
+/// /Heap  +--------+--------+--------+
```

Comment:
I'm not sure what the `/Heap` here means.

Reply:
`Stack/Heap` I just moved it down to save space, not sure how we can make it nicer.

Reply:
Oooh. That makes sense, thanks. Maybe leave out the entire `Stack/Heap` part and just have the table? If you want to bring more information across, I think that should be done in the surrounding text.

Reply:
I added this.